### PR TITLE
Feature/add basic UI

### PR DESCRIPTION
**Pull request by jaferIdrees**
### In this branch, I did the below tasks:

- [ ] Added a new file main.rb
- [ ] Added state class to store the new initiated objects.
- [ ] Added the main menu tasks with the related functionality

### CODE REVIEW

File: app/main.rb [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    puts 'Select a person from the following list by number'
+    list_people(indexed: true)
+    person = gets.chomp
+
+    print 'Date:'
+    date = gets.chomp
+    Rental.new(date, @books[book.to_i], @people[person.to_i])
+
+    'Rental created successfully'
+  end
```
Comment (andersonlebon) [inline app/main.rb:89]: - [ ] Kindly make sure your option(5) for creating rentals is working, Kindly make sure you fix the scenario when the user selects a person with the 0, your app should not exit rather it should create the appropriated rental.

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @jaferIdrees 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/main.rb (hunk 1 above)
Comment (jaferIdrees) [reply app/main.rb:89]: Done

**Review by sumairq — APPROVED** [review]
### Hi @jaferIdrees , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**